padle/padle.py

```diff
@@ -739,23 +775,35 @@ async def filter(self, ctx):
         dbcog = await self.get_dbcog()
         mgraph = dbcog.database.graph
         final = []
-        for i in range(1, mgraph.max_monster_id):
+        for i in range(1, mgraph.max_monster_id + 1):
             monster = dbcog.get_monster(i)
             if monster is None:
                 continue
-            if(self.is_valid_padle(monster, mgraph)):
+            if self.is_valid_padle(monster, mgraph):
                 final.append(str(i))
         with open("result.txt", "w") as file:
             file.write(",".join(final))
         with open("result.txt", "rb") as file:
             await ctx.send("List of monsters:", file=discord.File(file, "result.txt"))
         await ctx.tick()
-        
-    def is_valid_padle(monster, mgraph):
-        nextTrans = mgraph.get_next_transform(monster)
-        prevTrans = mgraph.get_prev_transform(monster)
-        return (monster.name_en is not None and monster.on_na and
+
+    def is_valid_padle(self, monster, mgraph):
+        next_trans = mgraph.get_next_transform(monster)
+        prev_trans = mgraph.get_prev_transform(monster)
+        return (  # Monster is in NA
+                monster.name_en is not None and monster.on_na and
+                # No collab monsters
                 monster.series.series_type != 'collab' and
-                ((monster.sell_mp >= 50000 and (monster.superawakening_count > 1 or prevTrans is not None)) or 
-                 "Super Reincarnated" in monster.name_en) and not monster.is_equip and
-                nextTrans is None and monster.level >= 99)
+                # No equips
+                not monster.is_equip and
+                # No monsters with next transforms
```

Comment:
fyi this is leaving out like, the "final" stage jiraya and alexander (? i think alexander) - probably not a problem though

Reply:
meh i think the disguises will just be annoying to guess but i can manually add alexander if you want?

Reply:
nope i dont think it matters, just wanted to point it out